src/go/rpk/pkg/cli/cmd/cluster/health.go

```diff
@@ -93,8 +93,10 @@ func printHealthOverview(hov *admin.ClusterHealthOverview) {
 	out.Section("CLUSTER HEALTH OVERVIEW")
 	overviewFormat := `Healthy:               %v
 Controller ID:         %v
+All nodes:             %v
 Nodes down:            %v
 Leaderless partitions: %v
 `
-	fmt.Printf(overviewFormat, hov.IsHealthy, hov.ControllerID, hov.NodesDown, hov.LeaderlessPartitions)
+	fmt.Printf(overviewFormat, hov.IsHealthy, hov.ControllerID, hov.AllNodes,
+		hov.NodesDown, hov.LeaderlessPartitions)
```

Comment:
I am not sure if there is some go formatter I should be using.

Reply:
@travisdowns `task rpk:fmt`

Reply:
Cool, it ran clean so I guess I got lucky.

Reply:
fwiw rpk:fmt doesn't seem to be terribly strict and i've seen feedback like manually using the format:

fmt.Printf("....",
    arg1,
    arg2,
    ...)

Reply:
I'd squash this into one line, it's more idiomatic.
The old formatter (which was changed to gofumpt) arbitrarily broke arguments at a specific number of columns, but that was often awkward.
The two most common approaches in Go is everything on one line, or once that becomes long, one arg per line.
Sometimes you see key,value dedicated on each line, or some other format for more complex use cases, but this one here is probably most standard with one line.

It's a bit hard to enforce a specific format given the various possibilities and reason for when to use them, though.

Reply:
Thanks for the suggestion and history. It's all on one line now.